frontend/src/pages/RunDetails.tsx

```diff
@@ -319,6 +321,7 @@ class RunDetails extends Page<RunDetailsInternalProps, RunDetailsState> {
                               <MD2Tabs
                                 tabs={[
                                   'Input/Output',
+                                  'Detail',
```

Comment:
nit: I think usually we call the tab `Details`
what do you think?

Reply:
That makes sense, I have updated the `details` term.